tests/unit/withOnyxTest.tsx

```diff
@@ -742,15 +756,15 @@ describe('withOnyxTest', () => {
 
     it('works with selectors', async () => {
         const selector = jest.fn().mockImplementation((value) => (value ? value.hello : undefined));
-        const TestComponentWithOnyx = withOnyx({
+        const TestComponentWithOnyx = withOnyx<ViewWithTextProps, {text: unknown}>({
             // Note: the prop passed to the wrapped component is called "text",
             // which is different than the key selected by the selector "hello"
             text: {
                 key: ONYX_KEYS.TEST_KEY,
                 selector,
             },
         })(ViewWithText);
-        let sourceData = {hello: 'world', goodnight: 'moon'};
+        let sourceData: unknown = {hello: 'world', goodnight: 'moon'};
```

Comment:
```suggestion
        let sourceData: OnyxValue<typeof ONYX_KEYS.TEST_KEY> = {hello: 'world', goodnight: 'moon'};
```

Reply:
It's actually the same as `OnyxValue<typeof ONYX_KEYS.TEST_KEY>` === `unknown`. I agree it looks better